src/Kuzzle.js

```diff
@@ -353,7 +361,7 @@ Kuzzle.prototype.connect = function () {
 
     connectionError.internal = error;
     self.state = 'error';
-    self.emitEvent('error', connectionError);
+    self.emitEvent('kerror', connectionError);
```

Comment:
`error` events have a special behaviour in Node, and throw an error if it is triggered with no listener (see https://nodejs.org/api/events.html#events_error_events )
So we need another event name for Kuzzle errors to avoid bad behaviour.

Looking for a better name, any suggestion is welcome ;)